rosa_cluster_admin/rosa_monitoring/rosa-accessing-monitoring-for-user-defined-projects.adoc

```diff
@@ -17,7 +15,9 @@ The `dedicated-admin` user has default permissions to configure and access monit
 Custom Prometheus instances and the Prometheus Operator installed through Operator Lifecycle Manager (OLM) can cause issues with user-defined project monitoring if it is enabled. Custom Prometheus instances are not supported.
 ====
 
+You can optionally disable monitoring for user-defined projects during or after a cluster installation.
```

Comment:
I'd move "optionally" to the beginning of the sentence:
"Optionally, you can disable..."

Reply:
@nalhadef thank you for your review and for this suggestion. I have subsequently applied your suggestion in this file and in `osd_cluster_admin/osd_monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc`.